index.html

```diff
@@ -4201,6 +4207,458 @@ <h3>Device Theft and Impersonation</h3>
     </section>
   </section>
 
+  <section class="appendix">
+    <h2>Proof Format Trade-Offs</h2>
+
+    <p>
+The Verifiable Credentials Data Model is designed to be compatible with a
+variety of existing and emerging syntaxes and digital proof formats. Each
+format has benefits and drawbacks. The following table is intended to summarize
+a number of these native trade-offs.
+    </p>
+
+    <p>
+The table below compares three proof format ecosystems; JSON+JWTs,
+JSON-LD+JWTs, and JSON-LD+LD-Proofs. At the time of publication of this
+specification, no organization is planning to use JSON+JWTs. The existence
+of JSON+JWTs in the table below is provided based on review comments on the
+specification for readers that were curious about the benefits that JSON-LD
+provided over JSON. Readers should also be aware that Zero-Knowledge Proofs
+are currently proposed as a sub-type of LD-Proofs and thus fall into the
+final colum below.
+    </p>
+
+    <table class="simple">
+      <thead>
+        <tr>
+          <th style="text-align: center;">Feature</th>
+          <th style="text-align: center;">JSON<br>+<br>JWTs</th>
+          <th style="text-align: center;">JSON&#8209;LD<br>+<br>JWTs</th>
+          <th style="text-align: center;">JSON&#8209;LD<br>+<br>LD&#8209;Proofs</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <tr>
+          <td>
+<a href="#pf1">PF1.</a> Support for open world data modelling.
+          </td>
+          <td class="missing">✖</td>
+          <td class="supported">✓</td>
+          <td class="supported">✓</td>
+        </tr>
+        <tr>
+          <td>
+<a href="#pf2">PF2.</a> Universal identifier mechanism for JSON objects via
+the use of URIs.
+          </td>
+          <td class="missing">✖</td>
+          <td class="supported">✓</td>
+          <td class="supported">✓</td>
+        </tr>
+        <tr>
+          <td>
+<a href="#pf3">PF3.</a> A way to disambiguate keys shared among different
+JSON documents by mapping them to IRIs via a context.
+          </td>
+          <td class="missing">✖</td>
+          <td class="supported">✓</td>
+          <td class="supported">✓</td>
+        </tr>
+        <tr>
+          <td>
+<a href="#pf4">PF4.</a> A mechanism to refer to data in an external document,
+where the data may be merged with the local document without a merge conflict
+in semantics or structure.
+          </td>
+          <td class="missing">✖</td>
+          <td class="supported">✓</td>
+          <td class="supported">✓</td>
+        </tr>
+        <tr>
+          <td>
+<a href="#pf5">PF5.</a> The ability to annotate strings with their language.
+          </td>
+          <td class="missing">✖</td>
+          <td class="supported">✓</td>
+          <td class="supported">✓</td>
+        </tr>
+        <tr>
+          <td>
+<a href="#pf6">PF6.</a> A way to associate datatypes with values such as
```

Comment:
JWT since it uses JSON also does this. There are standard fields within JWT that express timestamps as integers. It depends on the application like so much else.

Reply:
Yes, JWTs have that, but only for certain fields... and the datatype is hard coded. JSON-LD enables you to state that "in this context, interpret the value of `foo` as a Date" without hardcoding that into a specification. It's dynamic and machine readable. JSON has no such feature.

I've attempted to clarify the language and the meaning in 89a0645b9e291ff027359d94dfd060f5175a551d.